core/arch/aarch64/emit_utils.c

```diff
@@ -199,7 +199,8 @@ insert_exit_stub_other_flags(dcontext_t *dcontext, fragment_t *f, linkstub_t *l,
         /* mov x0, ... */
         pc = insert_mov_imm(pc, DR_REG_X0, (ptr_int_t)l);
         num_nops_needed = 4 - (pc - write_stub_pc - 1);
-        uint target_pc_slot_offs = (uint *)get_target_pc_slot(f, stub_pc) - pc;
+        uint *target_pc_slot = (uint *)get_target_pc_slot(f, stub_pc);
+        uint target_pc_slot_offs = target_pc_slot - pc;
```

Comment:
Maybe ASSERT(pc < target_pc_slot) so it's clear there's no underflow

Reply:
Done.